docs/introduction/QuickStart.mdx

```diff
@@ -0,0 +1,337 @@
+```
+
+</TabItem>
+
+<TabItem value="yarn">
+
+```bash
+yarn add @reduxjs/toolkit react-redux
+```
+
+</TabItem>
+
+</Tabs>
+
+#### Create an Online Project with CodeSandbox
+
+[CodeSandbox.io](https://codesandbox.io) is an online IDE that lets you create entire web applications in your browser, without needing to
+install anything locally.
+
+Create a new React project sandbox by browsing to the [CodeSandbox "Create Sandbox" page](https://codesandbox.io/s/) and selecting the official React template.
+
+Once the sandbox has been created, click the "Add Dependency" button on the left, search for `@reduxjs/toolkit`, add it, and then do the same for `react-redux`.
+
+### Create a Redux Store
+
+The React project starts with a single `<App>` component that shows some "hello" text on the page. Our first goal is to create a Redux store instance
+that holds some data inside, and update the `<App>` component to read that data and show it instead.
+
+<Note type="success">
+
+#### Key Concept: "Store"
+
+A Redux **_store_** is a special object that holds some data inside, such as a list of users that was fetched from the server.  
+The data inside the store object is called **_state_**.
+
+Each Redux store has three functions that you can call to interact with it:
+
+- `store.getState()` returns the current state value inside the store
+- `store.dispatch()` lets you tell the store that something has happened, and it should calculate a new state value in response
+- `store.subscribe()` lets you run some code whenever the store has been updated
+
+</Note>
+
+To create a Redux store, start by updating the `index.js` file to import the `configureStore` function from the `@reduxjs/toolkit` package.
+
+```js {3}
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { configureStore } from '@reduxjs/toolkit'
+
+import App from './App'
+
+const rootElement = document.getElementById('root')
+ReactDOM.render(<App />, rootElement)
+```
+
+In order to actually create a Redux store, we need to define our first "reducer" function.
+
+<Note type="success">
+
+#### Key Concept: "Reducers"
+
+A **_reducer_** is a function that know how to calculate a new state value when asked. A Redux store requires a single reducer function
+when it is created, which we call the "root reducer" to say that it is the starting point for any other reducer functions inside.
+
+Every reducer is called with two arguments: the current `state`, and an object called an **`action`**. We'll look at actions a bit later.
+
+Reducers must always follow certain rules inside. As a quick summary:
+
+- Reducers must always return a state value, never `undefined`
+- Reducers can only decide what the next state value should be by looking at the `state` and `action` arguments
+- Reducers are not allowed to make changes to the `state` argument, or anything else. They can only create the new state value by making copies of the originals, and updating the copies.
```

Comment:
This might be confusing to Redux Toolkit users (and we recommend it to beginners), as mutations are supported via Immer and are recommended in the docs.

Reply:
Yeah this is a good point. Later on, it would be good to show "here's the hard way (ignoring that immer is installed)" and "here's the easy way because of immer".

Reply:
Yep, this is still my #1 concern with trying to explain use of RTK in general, and especially in this page.